strings/names/psyphoza_first.txt

```diff
@@ -0,0 +1,8 @@
+Chanterelle
+Agaricus
+Ganoderma
+Lactarious
+Russula
+Sarcodon
+Gomphus
+Leccinum
```

Comment:
We could use some more random names here.
I suggest creating thread on forums or if you want, I can reach out to headmins so they can prepare naming guidelines and some names after this is approved.